ultralytics/data/split_dota.py

```diff
@@ -269,7 +268,7 @@ def split_test(data_root, save_dir, crop_size=1024, gap=200, rates=[1.0]):
     save_dir = Path(save_dir) / "images" / "test"
     save_dir.mkdir(parents=True, exist_ok=True)
 
-    im_dir = Path(os.path.join(data_root, "images/test"))
+    im_dir = Path(data_root) / "images" / "test"
     assert im_dir.exists(), f"Can't find {im_dir}, please check your data root."
     im_files = glob(str(im_dir / "*"))
```

Comment:
This too could be replaced with `list(im_dir.glob("*"))` I double checked, and turns out that `PIL.Image` will open `pathlib` objects directly.